Source/Views/StackViewController.swift

```diff
@@ -69,4 +75,80 @@ final class StackViewController: UIViewController {
         stackView.addArrangedSubview(childViewController.view)
         childViewController.didMove(toParentViewController: self)
     }
+    
+    /// A method to access the scrollView from outside and scroll to a given target ViewController. 
+    /// If no relativePosition parameter is provided, it will try to center the target.
+    ///
+    /// - Parameters:
+    ///   - target: The ViewController which should scroll into focus
+    ///   - relativePosition: An optional parameter to specify the alignement of the target. Depending of
+    ///     the stackViews' orientation some relativePosition values have different meaning: 
+    ///     beginning - top/left, end - bottom/right
+    public func scrollToViewController(target targetViewController: UIViewController,
```

Comment:
@ChristianOrgler I will adapt it, to use your naming, thx.
Why should the default be to animate it? - The only reason for no animation for me was to be more defensive: Animation is additional functionality. I am happy to set it to `true` if there's a valid reason.